pull_data.php

```diff
@@ -1,11 +1,9 @@
 <?php
 
-
 $db = new PDO('mysql:host=127.0.0.1;dbname=portfolio', 'root');
 
 $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE,PDO::FETCH_ASSOC);
 
-
```

Comment:
Oka, I canny be bothered to right Line break everywhere. Please get rid off unnecessary line breaks.